packages/visual-dev/src/components/Select.tsx

```diff
@@ -0,0 +1,90 @@
+// @ts-nocheck
```

Comment:
Why `@ts-nocheck`?

Reply:
Broken typing from `react-select` I'll replace it with a more specific and documented `@ts-ignore`.